django_project/lesson/templates/worksheet/detail.html

```diff
@@ -180,7 +180,7 @@ <h4><strong>
         </div>
         {% if worksheet.exercise_image %}
             <a href="#" class="pop-image">
-                <img id="{{ worksheet.exercise_image }}" class="img-responsive img-rounded pull-right"
+                <img id="/media/{{ worksheet.exercise_image }}" class="img-responsive img-rounded pull-right"
```

Comment:
Sorry, I still have notifications from some repositories ;-)
What about `{{ worksheet.exercise_image.url }}` so if you don't use `media/` anymore one day, nothing is broken.